eo-maven-plugin/src/main/java/org/eolang/maven/AssembleMojo.java

```diff
@@ -203,14 +203,15 @@ public void exec() throws IOException {
             new Moja<>(PlaceMojo.class),
         };
         while (true) {
+            final StopWatch times = new StopWatch();
             for (final Moja<?> moja : mojas) {
                 moja.copy(this).execute();
             }
             final String after = this.status();
             ++cycle;
             Logger.info(
-                this, "Assemble cycle #%d (%s -> %s)",
-                cycle, before, after
+                this, "Assemble cycle #%d (%s -> %s), took %.4f sec",
```

Comment:
@mximp you can use `"%[ms]s"` format from `Logger` -- it will print time nicely. see https://log.jcabi.com/decors.html

Reply:
Didn't know about that facility - done.